tests/infomax.cc

```diff
@@ -0,0 +1,53 @@
+#include "common.h"
```

Comment:
If you want to run these automatically from Jenkins you can get the google tests to emit 'junit':
```bash
tests --gtest_output="xml:test_results.xml"
````
And then add a stage like this to the Jenkins file
```groovy
buildStep("Gathering test results (" + env.NODE_NAME + ")") {
    dir("genn/tests") {
         // Process JUnit test output
         junit "**/test_results*.xml";
     }
}
```

Reply:
This is really cool! I might attempt this in a separate PR. We could probably also do with building things in ``tools`` and ``projects`` as part of our CI.